mailbox/store/src/main/java/org/apache/james/mailbox/store/search/comparator/CombinedComparator.java

```diff
@@ -51,35 +51,35 @@ public static CombinedComparator create(List<Sort> sorts) {
     private static Comparator<MailboxMessage> toComparator(Sort sort) {
```

Comment:
Easy search comparator are covered by sorts of Scrolling search testing, which is running our pretty complete search test suite.